src/platform/test/functional/services/data_views.ts

```diff
@@ -117,6 +117,10 @@ export class DataViewsService extends FtrService {
    * Switch Data View from top search bar
    */
   public async switchTo(name: string) {
+    // TODO: remove in https://github.com/elastic/kibana/issues/239313
+    if (await this.testSubjects.exists('nav-tour-skip-button')) {
+      await this.testSubjects.click('nav-tour-skip-button');
```

Comment:
We could also add `await this.testSubjects.waitForDeleted('nav-tour-skip-button');`